extension/scripts/features/displaycaseworth/ttDisplayCaseWorth.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+(async () => {
+	const feature = featureManager.registerFeature(
+		"Display Case Worth",
+		"display case",
+		() => settings.pages.displayCase.worth,
+		xhrListener,
+		addWorth,
+		removeWorth,
+		{
+			storage: ["settings.pages.displayCase.worth"],
+		},
+		() => {
+			if (!hasAPIData() || !settings.apiUsage.user.displaycase) return "No API access.";
```

Comment:
API Usage is meant for personal requests.
If they want to disable that call on others, they'll have to disable this feature.